### ISSUE TITLE
Rule Idea: console.log using spaces in arguments

### ISSUE THREAD

**Issue by vilchik-elena** [SonarSource/SonarTS#739]
These 2 lines doing the same thing, while second line should be preferred
```ts
console.log("Hello " + name);
console.log("Hello", name);
```

**Comment by alexeieleusis** [comment]
Sounds like what you want to do id forbid the + operator when the left hand is a string. What about string interpolation? Is it allowed? Preferred?

**Comment by vilchik-elena** [comment]
There is tslint `prefer-template` which forces developer to prefer string interpolation over concatenation. 
This rule is only about string concatenation inside `console` methods. Indeed, one might prefer `console.log("Hello ${name}");` to `console.log("Hello", name);`, but that's a matter of taste and btw there is property `allow-single-concat` to `prefer-template`.

I also thinking about case like that which might be considered as noncompliant for this rule: `console.log("Hello ", name);`. There will be 2 spaces in output.

Note I updates first comment with more examples
